### PR TITLE
re #4 select2 tags attribute

### PR DESCRIPTION
**Pull request by rmdstudio**
This PR allows us to exploit the tags attribute in the select2 lists. The tags attribute allows us to create new list items in a list.

### CODE REVIEW

**Comment by ercanozkaya** [comment]
@rmdstudio Not sure I understand why we need this. Why don't you just pass the tag attribute in the options array?

File: code/libraries/joomlatools/library/template/helper/listbox.php [hunk 1]
```diff
@@ -265,7 +268,10 @@ protected function _autocomplete($config = array())
             'deselect' => true,
         ))->append(array(
             'element'    => '#'.$config->attribs->id,
-            'options'    => array('multiple' => (bool) $config->attribs->multiple),
+            'options'    => array(
+                'multiple' => (bool) $config->attribs->multiple,
```
Comment (johanjanssens) [inline code/libraries/joomlatools/library/template/helper/listbox.php]: @ercanozkaya Why do we have the multiple in the attribs array ? It's passed as option, does it also have effect elsewhere ?

Comment (rmdstudio) [inline code/libraries/joomlatools/library/template/helper/listbox.php]: This PR is not needed. @ercanozkaya showed me how to pass the option. My bad. We close this PR now.

**Comment by johanjanssens** [comment]
Closing as invalid.

**Pull request closed without merging**